test/auth/features/oidc/broker.go

```diff
@@ -101,27 +114,40 @@ func BrokerSendEventWithOIDCTokenToDLS() *feature.Feature {
 	// Install DLS sink
 	f.Setup("install dead letter sink", eventshub.Install(dls,
 		eventshub.OIDCReceiverAudience(dlsAudience),
-		eventshub.StartReceiver))
-
-	// Install broker with DLS config
-	brokerConfig := append(
-		broker.WithEnvConfig(),
-		delivery.WithDeadLetterSinkFromDestination(&duckv1.Destination{
-			Ref:      service.AsKReference(dls),
-			Audience: &dlsAudience,
-		}),
-	)
-	f.Setup("install broker", broker.Install(brokerName, brokerConfig...))
+		eventshub.StartReceiverTLS))
+
+	f.Setup("install broker", func(ctx context.Context, t feature.T) {
+		brokerConfig := append(broker.WithEnvConfig(),
+			delivery.WithDeadLetterSinkFromDestination(&duckv1.Destination{
+				Ref:      service.AsKReference(dls),
+				Audience: &dlsAudience,
+				//CACerts: eventshub.GetCaCerts(ctx),
+			}))
+		broker.Install(brokerName, brokerConfig...)(ctx, t)
+	})
+
 	f.Setup("Broker is ready", broker.IsReady(brokerName))
 
+	// FIXME: current progress left over here. Need to figure out why trigger cannot be initialized correctly.
 	// Install Trigger
-	f.Setup("install trigger", trigger.Install(triggerName, brokerName,
-		trigger.WithSubscriber(nil, "bad://uri")))
+	//f.Setup("install trigger", trigger.Install(triggerName, brokerName,
+	//	trigger.WithSubscriber(nil, "bad://uri")))
+
+	f.Setup("install the trigger and specify the CA cert of the destination", func(ctx context.Context, t feature.T) {
+		// create an empty destination ref
+		d := service.AsDestinationRef("")
+		d.CACerts = eventshub.GetCaCerts(ctx)
+		// uri is an addressable, create a new one and put the bad	uri in it
+		d.URI, _ = apis.ParseURL("bad://uri")
+		trigger.Install(triggerName, brokerName, trigger.WithSubscriberFromDestination(d))(ctx, t)
```

Comment:
I got a `failed to create resource admission webhook "validation.webhook.eventing.knative.dev" denied the request: validation failed: Absolute URI is not allowed when Ref or [apiVersion, kind, name] is present: spec.subscriber.ref, spec.subscriber.uri, spec.subscriber[apiVersion, kind, name]` when running this.
So there is no need to create a destination via `service.AsDestinationRef("")`. You can create it directly via `duckv1.Destination{}` and propagate the fields then.